packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/form-action/components/WorkflowEditActionForm.tsx

```diff
@@ -169,13 +202,17 @@ export const WorkflowEditActionForm = ({
                         return;
                       }
 
+                      const updatedFormData = formData.filter(
+                        (f) => f.id !== field.id,
```

Comment:
nitpick: the same thing about the `f` variable

I thought we had an ESLint rule for these cases. It's not the most interesting thing to discuss in a PR 😅